index.html

```diff
@@ -124,7 +124,7 @@ <h1 id="title">RDF-star Working Group Charter</h1>
               Chairs
             </th>
             <td>
-              <i class="todo">[chair name] (affiliation)</i>
+              Ora Lassila (Amazon), <i class="todo">[chair name] (affiliation)</i>
```

Comment:
FYI: Material from PR #30.

Reply:
Oops :-( Thanks for pointing that out...